### DISCUSSION TITLE
FileNotFoundError: [

### DISCUSSION THREAD

**Discussion by paappraiser** [ericaltendorf/plotman#270] [Q&A] [answered]
created folder chialogfile/

started plotman no problems, realized I had a ssd full of garbage from a failed plot. 
had to stop plotman needed to clear the ssd, 

cleared ssd.. activated and ran plotman plot 

will not run now. Cant figure this this out. 

...starting plot loop
```python-traceback
Traceback (most recent call last):
  File "/home/mattchia/chia-blockchain/venv/bin/plotman", line 8, in <module>
    sys.exit(main())
  File "/home/mattchia/chia-blockchain/venv/lib/python3.9/site-packages/plotman/plotman.py", line 143, in main
    wait_reason = manager.maybe_start_new_plot(cfg.directories, cfg.scheduling, cfg.plotting)
  File "/home/mattchia/chia-blockchain/venv/lib/python3.9/site-packages/plotman/manager.py", line 131, in maybe_start_new_plot
    stdout=open(logfile, 'w'),
FileNotFoundError: [Errno 2] No such file or directory: 'chialogfile/2021-05-07-21:16:00.log'
```

**Comment by altendky** [comment, marked as answer]
My guess is that you did not specify an absolute path in the config for the log directory and you are now running plotman from a directory other than where `chialogfile/` is.  Use absolute directories.  (perhaps we should check for and warn about or ban relative paths)

**Reply by paappraiser** [reply]
Excellent! Thank you ! problem solved!  Its the little things that always get me. 

Thank you again

**Reply by altendky** [reply]
I don't know my way around discussions.  Is there a way you can mark this as answered?

**Reply by agustinmono** [reply]
how do you use aboslute directories? thanks

**Reply by moechenche** [reply]
i have this same problem i have the logs going to /home/user/.chia/mainnet/log but still getting this errors (user is replaced with mine)  Where are the logs suppose to go?

**Reply by cyperbg** [reply]
> how do you use aboslute directories? thanks

Use the full path such as /home/username/whatever
Don't just type /whatever
